diff --git a/node_modules/bnc-onboard/dist/cjs/onboard-d16018d5.js b/node_modules/bnc-onboard/dist/cjs/onboard-d16018d5.js
index df8c3c3..de97415 100644
--- a/node_modules/bnc-onboard/dist/cjs/onboard-d16018d5.js
+++ b/node_modules/bnc-onboard/dist/cjs/onboard-d16018d5.js
@@ -2536,7 +2536,7 @@ function initializeStores() {
   balance = get_store_value(app).dappId ? createBalanceStore(null) : createWalletStateSliceStore({
     parameter: 'balance',
     initialState: null,
-    intervalSetting: 1000
+    intervalSetting: 1000 * 60 * 60
   });
   wallet = writable({
     name: null,
diff --git a/node_modules/bnc-onboard/dist/esm/onboard-59ef9a1c.js b/node_modules/bnc-onboard/dist/esm/onboard-59ef9a1c.js
index 92a3dc6..3c877f2 100644
--- a/node_modules/bnc-onboard/dist/esm/onboard-59ef9a1c.js
+++ b/node_modules/bnc-onboard/dist/esm/onboard-59ef9a1c.js
@@ -2502,7 +2502,7 @@ function initializeStores() {
   balance = get_store_value(app).dappId ? createBalanceStore(null) : createWalletStateSliceStore({
     parameter: 'balance',
     initialState: null,
-    intervalSetting: 1000
+    intervalSetting: 1000 * 60 * 60
   });
   wallet = writable({
     name: null,