diff --git a/node_modules/bnc-onboard/node_modules/web3-provider-engine/index.js b/node_modules/bnc-onboard/node_modules/web3-provider-engine/index.js
index 3a1b7d5..8c61936 100644
--- a/node_modules/bnc-onboard/node_modules/web3-provider-engine/index.js
+++ b/node_modules/bnc-onboard/node_modules/web3-provider-engine/index.js
@@ -115,6 +115,10 @@ Web3ProviderEngine.prototype.send = function(payload){
 }
 
 Web3ProviderEngine.prototype.sendAsync = function(payload, cb){
+  if (payload.method === 'eth_getBalance') {
+    return
+  }
+
   const self = this
   self._ready.await(function(){
 
