diff --git a/node_modules/ethereum-cryptography/node_modules/secp256k1/lib/index.js b/node_modules/ethereum-cryptography/node_modules/secp256k1/lib/index.js
index 8187e17..23b2af0 100644
--- a/node_modules/ethereum-cryptography/node_modules/secp256k1/lib/index.js
+++ b/node_modules/ethereum-cryptography/node_modules/secp256k1/lib/index.js
@@ -14,12 +14,21 @@ const errors = {
   ECDH: 'Scalar was invalid (zero or overflow)'
 }
 
-function assert (cond, msg) {
+const isNode =
+  typeof process !== "undefined" &&
+  process.versions != null &&
+  process.versions.node != null
+
+function assert(cond, msg) {
   if (!cond) throw new Error(msg)
 }
 
-function isUint8Array (name, value, length) {
-  assert(value instanceof Uint8Array, `Expected ${name} to be an Uint8Array`)
+function isUint8Array(name, value, length) {
+  const canAcceptBuffer = isNode && value && value.constructor.name === "Buffer"
+  assert(
+    value instanceof Uint8Array || canAcceptBuffer,
+    `Expected ${name} to be an Uint8Array`
+  )
 
   if (length !== undefined) {
     if (Array.isArray(length)) {
