diff --git a/node_modules/secp256k1/lib/index.js b/node_modules/secp256k1/lib/index.js
index 8187e17..ab31b98 100644
--- a/node_modules/secp256k1/lib/index.js
+++ b/node_modules/secp256k1/lib/index.js
@@ -19,7 +19,11 @@ function assert (cond, msg) {
 }
 
 function isUint8Array (name, value, length) {
-  assert(value instanceof Uint8Array, `Expected ${name} to be an Uint8Array`)
+  const valueIsBuffer = !process.browser && value && value.constructor.name === "Buffer"
+  assert(
+    value instanceof Uint8Array || valueIsBuffer,
+    `Expected ${name} to be an Uint8Array`
+  )
 
   if (length !== undefined) {
     if (Array.isArray(length)) {
